packages/components/credentials/Mailchimp/Mailchimp.ts

```diff
@@ -0,0 +1,25 @@
+import { INodeParams, INodeCredential } from '../../src/Interface'
+class MailchimpApi implements INodeCredential {
+    name: string
+    version: number
+    credentials: INodeParams[]
+    constructor() {
+        this.name = 'mailChimpCredential'
+        this.version = 1.0
+        this.credentials = [
+            {
+                label: 'Mailchimp API Key',
+                name: 'apiKey',
+                type: 'string',
+                default: ''
+            },
+            {
+                label: 'Mailchimp data center',
```

Comment:
we dont need data center field here. From documentation - It’s also appended to your API key in the form key-dc; if your API key is 0123456789abcdef0123456789abcde-us6, then the data center subdomain is us6.